### PR TITLE
Issues/2297 update dependencies

### PR DESCRIPTION
**Pull request by bencvdb**
Update Toil dependencies.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,9 +20,9 @@ def runSetup():
     Calls setup(). This function exists so the setup() invocation preceded more internal
     functionality. The `version` module is imported dynamically by importVersion() below.
     """
-    boto = 'boto==2.38.0'
-    boto3 = 'boto3>=1.4.7, <2.0'
-    futures = 'futures==3.1.1'
+    boto = 'boto==2.48.0'  # Previously 'boto==2.38.0'
+    boto3 = 'boto3>=1.7.50, <2.0'  # Previously 'boto3==1.4.7'
+    futures = 'futures==3.2.0'  # Previously 'futures==3.1.1'
```
Comment (DailyDreaming) [inline setup.py]: Keep this as `futures==3.1.1` since that is the latest version in python 3.

Comment (bencvdb) [reply setup.py]: Its been reverted

**Comment by bencvdb** [comment]
@jessebrennan I think it was causing tests to fail.

**Comment by DailyDreaming** [comment]
Finished tests will be here: http://jenkins.cgcloud.info/job/toil-pull-requests/4217

**Comment by DailyDreaming** [comment]
Jenkins, test this please.

**Comment by DailyDreaming** [comment]
New test results will be here:
http://jenkins.cgcloud.info/job/toil-pull-request-integration-tests/2502
http://jenkins.cgcloud.info/job/toil-pull-requests/4220

**Comment by DailyDreaming** [comment]
Jenkins, test this please.

**Pull request merged**